views/docs/console.md

```diff
@@ -52,6 +52,22 @@ Note that if you are using Asterisk, the format is slightly different. The examp
 AHN> Adhearsion::OutboundCall.originate 'SIP/user1'
 </pre>
 
+As well, if you need to place calls out into the outside world, you may get an error message 
+such as "I don't know how to authenticate..." if you try to call directly. 
```

Comment:
Ah.. that's better.... let me update.

On Wed, Nov 11, 2015 at 12:48 PM, Ben Klang <notifications@github.com>
wrote:

> In views/docs/console.md
> <https://github.com/adhearsion/adhearsion-website/pull/37#discussion_r44562073>
> :
>
> > @@ -52,6 +52,22 @@ Note that if you are using Asterisk, the format is slightly different. The examp
> >  AHN> Adhearsion::OutboundCall.originate 'SIP/user1'
> >  </pre>
> >
> > +As well, if you need to place calls out into the outside world, you may get an error message
> > +such as "I don't know how to authenticate..." if you try to call directly.
>
> So this is where we get into a grey area. The problem you're seeing comes
> from the fact that you dialed an IAX URI, rather than a preconfigured peer.
> If you had a peer definition in iax.conf and called it by name (eg, dial
> "IAX2/4443@peer"), you would not have received that error. The same would
> be true of SIP, though it's very probable that the error message is
> different. My feeling on this is that we either need to go into
> (potentially a lot) more detail regarding best practices for Asterisk
> config, or we need to refer the reader to documentation that explains these
> nuances. I lean toward the latter, if it can be found.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/adhearsion/adhearsion-website/pull/37/files#r44562073>
> .
>